openwisp_users/api/tests/test_authentication.py

```diff
@@ -0,0 +1,30 @@
+from django.core.cache import cache
+from django.test import RequestFactory, TestCase
+from openwisp_users.api.authentication import BearerAuthentication
+from openwisp_users.tests.utils import TestMultitenantAdminMixin
+from rest_framework.decorators import api_view, authentication_classes
+from rest_framework.response import Response
+
+from . import AuthenticationMixin
+
+
+class AuthenticationTests(TestMultitenantAdminMixin, AuthenticationMixin, TestCase):
```

Comment:
for consistency with the other modules, we should move these tests to `openwisp_users.tests.test_api`.

I'd merge the 3 classes into one since there's not many lines in each class so one file and one class for all these should be well manageable